### PR TITLE
Install `oc` and bump to `stable-4.16` version

### PR DESCRIPTION
**Pull request by dhaiducek**

### CODE REVIEW

**Review by yiraeChristineKim — DISMISSED** [review]

File: build/download-clis.sh [hunk 1]
```diff
@@ -5,16 +5,17 @@
 set -e
 
 CLI_DESTINATION_DIR="${CLI_DESTINATION_DIR:=/usr/local/bin}"
+CLI_VERSION=4.16
```
Comment (yiraeChristineKim) [inline build/download-clis.sh:8]: Do you have a reason 4.16 why not `latest`?

Comment (dhaiducek) [reply build/download-clis.sh:8]: Specifying a channel will give us more stability while giving us the benefit of z-stream updates. Setting it to `latest` has the potential to grab a new version that's broken.

**Review by yiraeChristineKim — APPROVED** [review]

**Comment by dhaiducek** [comment]
/override ci/prow/test-e2e-grc-framework

The operator test failed, but it's not related to `oc`.

**Pull request merged**